api_app/tests_ma/test_db/test_repositories/test_workpaces_repository.py

```diff
@@ -201,3 +203,17 @@ def test_automatically_create_application_registration_returns_false(workspace_r
     dictToTest = {"client_id": "12345"}
 
     assert workspace_repo.automatically_create_application_registration(dictToTest) is False
+
+
+def test_workspace_owner_is_set_if_not_present_in_workspace_properties(workspace_repo):
+    dictToTest = {}
+    expected_object_id = "Expected"
+
+    assert workspace_repo.get_workspace_owner(dictToTest, expected_object_id) is expected_object_id
+
+
+def test_workspace_owner_is_not_overwritten_if_not_present_in_workspace_properties(workspace_repo):
+    dictToTest = {"workspace_owner_object_id": "Expected"}
+    not_expected_object_id = "Not Expected"
+
+    assert workspace_repo.get_workspace_owner(dictToTest, not_expected_object_id) == "Expected"
```

Comment:
I think it will be also good to have a positive test where you do have a workspace owner specified in the properties to make sure it is set correctly

Reply:
Maybe I misunderstood what you mean, but there are 2 new tests
1) Does not have `workspace_owner_object_id` in a dictionary (workspace properties), so therefore returns the default.
2) Does have `workspace_owner_object_id` in a dictionary (workspace properties), so ignores the default